genai-perf/genai_perf/inputs/retrievers/synthetic_prompt_generator.py

```diff
@@ -174,6 +174,10 @@ def _generate_prompt_with_token_reuse(
         """
         final_prompt: List[int] = []
         size_to_use = block_size
+        last_hash_length = num_tokens - ((len(prompt_hash_list) - 1) * block_size)
+        assert (
+            last_hash_length > 0 and last_hash_length <= block_size
+        ), f"Input_length: {num_tokens}, Hash_ids: {prompt_hash_list}, Block_size: {block_size} are not compatible."
```

Comment:
Looks good. Can you update this error test to specify the expected input_length range? And to make it clear what the formula for that range is?

Reply:
updated